.stylelintrc

```diff
@@ -2,6 +2,6 @@ extends: stylelint-config-sass-guidelines
 
 rules:
   max-nesting-depth: 3
-  selector-class-pattern: ^[a-z]+(?:[-|A-Z][a-z]+)*$
+  selector-class-pattern: ^[a-z]+(?:[A-Z|\-|\_]{1,2}[a-z]+)*$
```

Comment:
Ah, I reviewed the other PR before I saw this. 
But I guess that the latest regex pattern we want is the one from the Overlay conversion: `^[a-z0-9]+(?:[A-Z|\-|\_]{1,2}[a-z0-9]+)*$` when this gets merged in?

Reply:
Yep, I will resolve to later on. There were so many different cases, e.g. overriding some 3rd party dependency styles... But still did not want to let just _any_ class pattern.